doc/how-to/getting_started_go.rst

```diff
@@ -1,256 +1,279 @@
 Executing stored procedures
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To execute a stored procedure, use ``NewCallRequest()``:
+
+..  literalinclude:: /code_snippets/snippets/connectors/go/hello.go
+    :language: go
+    :start-at: // Call
+    :end-at: Stored procedure result
+    :dedent:
+
+
+.. _getting_started_go_closing_connection:
+
+Closing the connection
+~~~~~~~~~~~~~~~~~~~~~~
+
+The ``CloseGraceful()`` method can be used to close the connection when it is no longer needed:
+
+..  literalinclude:: /code_snippets/snippets/connectors/go/hello.go
+    :language: go
+    :start-at: // Close connection
+    :end-at: Connection is closed
+    :dedent:
+
+..  NOTE::
 
-Switch to the terminal window where Tarantool is running.
+    You can find the example with all the requests above on GitHub: `go <https://github.com/tarantool/doc/tree/latest/doc/code_snippets/snippets/connectors/go>`_.
 
-.. NOTE::
 
-    If you don't have a terminal window with remote connection to Tarantool,
-    check out these guides:
 
-    * :ref:`connecting to a local Tarantool instance <connecting-remotely>`
-    * :ref:`attaching to a Tarantool instance that runs in a Docker container <getting_started-docker-attaching>`
+.. _getting_started_go_run_client_app:
 
-Define a simple Lua function:
+Starting a client application
+-----------------------------
 
-.. code-block:: lua
+1.  Execute the following ``go get`` commands to update dependencies in the ``go.mod`` file:
 
-    function sum(a, b)
-        return a + b
-    end
+    .. code-block:: console
 
-Now we have a Lua function defined in Tarantool. To invoke this function from
-``go``, use ``Call``:
+        $ go get github.com/tarantool/go-tarantool/v2
+        $ go get github.com/tarantool/go-tarantool/v2/decimal
+        $ go get github.com/tarantool/go-tarantool/v2/uuid
 
-.. code-block:: go
+2.  To run the resulting application, execute the ``go run`` command in the application directory:
 
-    resp, err = conn.Call("sum", []interface{}{2, 3})
+    .. code-block:: console
 
-To send bare Lua code for execution, use ``Eval``:
+        $ go run .
+        Inserted tuples:
+        [[1 Roxette 1986]]
+        [[2 Scorpions 1965]]
+        [[3 Ace of Base 1987]]
+        [[4 The Beatles 1960]]
+        Tuple selected the primary key value: [[1 Roxette 1986]]
+        Tuple selected the secondary key value: [[4 The Beatles 1960]]
+        Updated tuple: [[2 Pink Floyd 1965]]
+        Replaced tuple: [[1 Queen 1970]]
+        Deleted tuple: [[5 The Rolling Stones 1962]]
+        Stored procedure result: [[[2 Pink Floyd 1965] [4 The Beatles 1960]]]
+        Connection is closed
 
-.. code-block:: go
 
-    resp, err = connection.Eval("return 4 + 5", []interface{}{})
 
 .. _getting_started-go-comparison:
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Feature comparison
```

Comment:
Need a clearer heading here. Readers don't know about other connectors **Feature comparison** is unclear in this context: "Whose features? Comparison with what?"